mod/ckeditor/views/default/elgg/ckeditor.js

```diff
@@ -13,13 +13,19 @@
 define(function (require) {
 	var elgg = require('elgg');
 	require('elgg/init');
+
 	var $ = require('jquery');
 	require('jquery.ckeditor');
+
 	var CKEDITOR = require('ckeditor');
-	var config = require('elgg/ckeditor/config');
 
 	var elggCKEditor = {
-		bind: function (selector) {
+		bind: function (selector, config) {
+			if (config === undefined) {
+				config = require('elgg/ckeditor/config');
```

Comment:
You *can* convert it to the async version: `require(['elgg/ckeditor/config'], function (default_config) {})`.